prooph/event-store-symfony-bundle/0.4/config/packages/prooph_event_store.yaml

```diff
@@ -8,3 +8,4 @@ services:
         public: false
 
     Prooph\EventSourcing\EventStoreIntegration\AggregateTranslator: null
+    Prooph\EventStore\EventStore: '@prooph_event_store.default'
```

Comment:
This configuration is brought by the `prooph/pdo-event-store` [flex recipe](https://github.com/symfony/recipes-contrib/blob/master/prooph/pdo-event-store/1.7/config/packages/prooph_pdo_event_store.yaml#L5)